oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java

```diff
@@ -24,7 +24,11 @@
   private final int maxLat;
 
   /**
-   * Creates an {@code OSHDBBoundingBox} instance from osm int coordinates.
+   * Creates an {@code OSHDBBoundingBox} instance from transposed coordinates.
+   * 
+   * This method is mainly for internal usage. Coordinates are stored as integer
+   * internally by multiplying and rounding them (see 
+   * {@code bboxWgs84Coordinates}) named osm-coordinates.
```

Comment:
```suggestion
   * Creates an {@code OSHDBBoundingBox} instance from scaled OSM coordinates.
   * 
   * <p>This method is mainly for internal usage. OSM stores coordinates with a fixed precision
   * of <a href="https://wiki.openstreetmap.org/wiki/Node#Structure">7 decimal digits</a> and
   * stores them internally as integers. You can use this method to create a bounding box from
   * such (scaled and rounded) coordinates.</p>
```

Reply:
thanks, this is much better, I adjusted it some more